src/RadixSortLSDClass.chpl

```diff
@@ -0,0 +1,225 @@
+/* Radix Sort Least Significant Digit */
+module RadixSortLSDClass {
+    use BlockDist;
+    use BitOps;
+    use AryUtil;
+    use CommAggregation;
+    use IO;
+    use CTypes;
+    use Reflection;
+    use Logging;
+    use ServerConfig;
+
+    private config const logLevel = ServerConfig.logLevel;
+    const rsLogger = new Logger(logLevel);
+
+    proc radixSortLSD_memEst(size: int, itemsize: int, numTasks: int = here.maxTaskPar, bitsPerDigit: int = RSLSD_bitsPerDigit) {
```

Comment:
move memEst routines into the class and make them use the class consts for tasks, etc.

Reply:
ah yeah I wasn't really thinking here. In my head, i was thinking we'd want to run the mem estimates before creating a `RadixSortLSDInstance`, but the memory overhead is in running the sort, not creating an instance of the class. I'll get those moved!